openshift/patches/fix_scheduler_and_autoscaler.patch

```diff
@@ -0,0 +1,332 @@
+diff --git a/vendor/knative.dev/eventing/pkg/scheduler/state/helpers.go b/vendor/knative.dev/eventing/pkg/scheduler/state/helpers.go
+index 56ca82b8d..b37fc7997 100644
+--- a/vendor/knative.dev/eventing/pkg/scheduler/state/helpers.go
```

Comment:
will this hit "upstream" (eventing core)? Or are we just on not "latest" version?